core/src/Pos/Core/Block/Constructors.hs

```diff
@@ -132,6 +131,21 @@ mkMainBlockExplicit pm bv sv prevHash difficulty slotId sk pske body =
             (mkAttributes ())
             (hash extraB)
 
+
+{-
+
+# mhueschen: this is not used anywhere in the codebase
```

Comment:
This bit is gross, and I'm not sure what to do about it. `emptyMainBody` isn't used anywhere, and it caused several of our offending dependencies. Removing it seemed reasonable, but I wanted to preserve it in case it's needed again.

I'm also unsure of the convention around leaving usernames in source files. I'm open to removing it.

Reply:
I'm not sure I'd keep this in here. However, we could wait for someone else to weigh in and see what they think.

Reply:
Yeah you can go ahead and delete this whole thing - it's in the `git` history anyway, so not like it's gone forever!

Reply:
Ok great, I'll delete it 👍